services/wfm/src/main/java/org/openkilda/wfm/topology/switchmanager/SwitchManagerTopologyConfig.java

```diff
@@ -30,4 +36,19 @@ default String getKafkaNorthboundTopic() {
     default String getKafkaSpeakerTopic() {
         return getKafkaTopics().getSpeakerTopic();
     }
+
+    @Key("flow.meter.burst.coefficient")
```

Comment:
there is no such option. we have only `burst.coefficient`

Reply:
Fixed.